libs/wingsdk/src/target-sim/counter.ts

```diff
@@ -45,6 +60,14 @@ export class Counter extends cloud.Counter {
     };
   }
 
+  /** @internal */
+  public _liftedFields(): Record<string, string> {
+    const env = makeEnvVarName("resource", this.backend);
+    return {
+      $handle: `process.env["${env}"]`,
+    };
+  }
```

Comment:
Guessing this can be shared by basically all sim resources.

Reply:
Eventually - though for now the implementations may look different until we've closed https://github.com/winglang/wing/issues/6471

Reply:
It looks like this was correct (I copied the code into a function named "simulatorLiftedFieldsFor")